src/accelerate/components/Hero.tsx

```diff
@@ -44,10 +52,38 @@ export default function Hero( props: Props ) {
 					{ !! props.period && ( props.canViewAnalytics || props.canViewInsights ) && (
 						<nav className='Hero__links'>
 							{ props.canViewAnalytics && (
-								<a href="index.php?page=altis-analytics">{ __( 'Analytics', 'altis-analytics' ) }</a>
+								<a
```

Comment:
One option could be to make a new component for links that has the tracking built in, so some props could be provided only once e.g. `url` or `href` etc... Not required but always something to consider when you spot some repetition in the code.

Reply:
Good thinking and totally agreed ✅